### PR TITLE
Fix the env path to include mpiexec

### PR DESCRIPTION
**Pull request by chrishkchris**
In dockerfile, this fix registers the mpiexec command in environment variable path automatically
Before this fix it needs to add the path of the mpi binary manually

### CODE REVIEW

**Comment by chrishkchris** [comment]
today, I am setting the environment for distributed training and test in this PR

**Comment by chrishkchris** [comment]
A test of the mpiexec command in dockerfile in single node (panda6)

root@de81766b25d7:~/dcsysh/singa/examples/autograd# mpiexec --hostfile host_file python3 mnist_dist.py
Starting Epoch 0:
Training loss = 463.196106, training accuracy = 0.836856
Evaluation accuracy = 0.967949, Elapsed Time = 2.289939s
Starting Epoch 1:
Training loss = 149.340302, training accuracy = 0.949519
Evaluation accuracy = 0.980469, Elapsed Time = 2.257896s
Starting Epoch 2:
Training loss = 112.977020, training accuracy = 0.961856
Evaluation accuracy = 0.971855, Elapsed Time = 2.304186s
Starting Epoch 3:
Training loss = 95.893265, training accuracy = 0.967481
Evaluation accuracy = 0.972556, Elapsed Time = 2.297554s
Starting Epoch 4:
Training loss = 84.966179, training accuracy = 0.971371
Evaluation accuracy = 0.981771, Elapsed Time = 2.238955s
Starting Epoch 5:
Training loss = 76.566475, training accuracy = 0.973858
Evaluation accuracy = 0.979267, Elapsed Time = 2.248186s
Starting Epoch 6:
Training loss = 73.179596, training accuracy = 0.975544
Evaluation accuracy = 0.984075, Elapsed Time = 2.247814s
Starting Epoch 7:
Training loss = 67.654587, training accuracy = 0.977597
Evaluation accuracy = 0.976863, Elapsed Time = 2.257856s
Starting Epoch 8:
Training loss = 62.395630, training accuracy = 0.978549
Evaluation accuracy = 0.981370, Elapsed Time = 2.267953s
Starting Epoch 9:
Training loss = 59.330917, training accuracy = 0.980335
Evaluation accuracy = 0.983674, Elapsed Time = 2.242991s

I will need to think about how to use docker to launch mpiexec in multi-node 

This PR is ready for merge
